generators/generator-constants.js

```diff
@@ -1,15 +1,25 @@
 'use strict';
 
-// all constants used through out all generators
+// all constants used throughout all generators
+
+const MAIN_DIR = 'src/main/';
+const TEST_DIR = 'src/test/';
+
 const constants = {
     QUESTIONS : 15,
-    RESOURCE_DIR : 'src/main/resources/',
-    WEBAPP_DIR : 'src/main/webapp/',
-    ANGULAR_DIR : 'src/main/webapp/app/',
```

Comment:
It isn't referenced anywhere in this script though? But sure, if you think it has purpose I'll put it back.